pages/projects_page.py

```diff
@@ -114,3 +144,20 @@ def is_workspace_found(self, workspace_name):
     @allure.step("Get projects' page title")
     def get_title(self):
         return self.get_text(self._PROJECT_PAGE_TITLE)
+
+    def get_no_project_found_msg(self):
+        return self.get_text(self._NO_PROJECT_FOUND_MSG)
+
+    @allure.step("Check if {project_name} is present")
+    def is_project_found(self, project_name):
+        projects_titles = self._wait.until(
+            expected_conditions.visibility_of_all_elements_located(self._PROJECTS_TITLES))
+        for project_title in projects_titles:
```

Comment:
maybe we can short the for using the index 
https://www.programiz.com/python-programming/methods/list/index

like return projects_titles.index('project_name') -> true or false of exists

Reply:
We can't perform this on list of webelement since the project_name is a string.
this will need us to create a local list that contains the text of each webelement in projects_titles